SU-SOE/soe_basic
# D8CORE-4020: Adding SOE styling for publication teasers

# NOT READY FOR REVIEW

# Summary
- Adding styles for the SOE subtheme

# Review By (Date)
- 3/4

# Urgency
- Med

# Steps to Test

1. Pull in this change.
2. Clear caches.
3. Add teasers on a page.
4. Verify it looks as design noted: 
"Yes, SOE should use their usual blue with underline and corresponding arrows for internal/external links. "
5. It will match what the News one looks like.

# Affected Projects or Products
- Teasers for publications

# Associated Issues and/or People
- D8CORE-4020
- https://github.com/SU-SWS/stanford_profile_helper/pull/94


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
